examples/imaging_demo.py

```diff
@@ -213,52 +197,68 @@
 ###############################################################################
 # Create a back projection image with natural weighting
 
-bp_nat = vis_to_image(stix_vis1, imsize, pixel_size=pixel)
+bp_nat = vis_to_image(vis10_3, imsize, pixel_size=pixel)
 
 ###############################################################################
 # Create a back projection image with uniform weighting
 
-bp_uni = vis_to_image(stix_vis1, imsize, pixel_size=pixel, scheme="uniform")
+bp_uni = vis_to_image(vis10_3, imsize, pixel_size=pixel, scheme="uniform")
 
 ###############################################################################
 # Create a `sunpy.map.Map` with back projection
 
-bp_map = vis_to_map(stix_vis1, imsize, pixel_size=pixel)
+bp_map = vis_to_map(vis10_3, imsize, pixel_size=pixel)
 
 ###############################################################################
-# Crete a map using the clean algorithm `vis_clean`
+# Crete a clean image using the clean algorithm `vis_clean`
 
 niter = 200  # number of iterations
 gain = 0.1  # gain used in each clean iteration
 beam_width = 20.0 * u.arcsec
 clean_map, model_map, resid_map = vis_clean(
-    stix_vis1, imsize, pixel_size=pixel, gain=gain, niter=niter, clean_beam_width=20 * u.arcsec
+    vis10_3, imsize, pixel_size=pixel, gain=gain, niter=niter, clean_beam_width=20 * u.arcsec
 )
 
+###############################################################################
+# Create a sunpy map for the clean image in Helioprojective
+
+header = make_fitswcs_header(
+    clean_map.data,
+    max_stix.transform_to(Helioprojective(obstime=vis_tr.center, observer=solo)),
+    telescope="STIX",
+    observatory="Solar Orbiter",
+    scale=pixel,
+    rotation_angle=90 * u.deg + roll,
+)
+
+clean_map = Map((clean_map.data, header))
```

Comment:
Why is the map created outside of clean? Is it possible to make it inside as for the other methods?

Reply:
Currently `vis_clean` makes a map assuming helioprojective which in this case is not correct.

Reply:
Essentially once TCDSolar/xrayvision/issues/56 is done issue is resolved this won't be necessary